### DISCUSSION TITLE
Build instructions and CMake configuration should be updated to reflect that devilutionX.mpq is now required

### DISCUSSION THREAD

**Discussion by efedo** [diasurgical/DevilutionX#3045] [General]
Now that devilutionX.mpq is required for font handling, build instructions should probably be updated to reflect this change, and an installation path/instructions for the smpq prerequisite should probably be added for platforms on which they are currently missing. It would also make sense to modify the default CMake configuration to make packaging of devilutionX.mpq non-optional, since it is actually required to run, otherwise the default configuration [will fail to run](https://github.com/diasurgical/devilutionX/issues/3043).

It may also be worth re-considering which community members are given permissions to close issues, since rude dismissals of legitimate issues are likely to deter new community members from contributing.

**Comment by qndel** [comment]
You opened a discussion so I can't close it huh? ;)

As for "rude community members", it's the first time I see you here and you are the rude one, in fact he was pretty polite towards you. Latest devilutionx.mpq will be provided with 1.3.0 release which happens this month - you can also download one of automated builds from github - they have it inside too. That's why this step is marked as optional.
Nobody is stopping you from following the build instructions (like you've been repeatedly told to) and getting it to build

**Comment by StephenCWills** [comment]
I personally take it as an insult to both me and the other maintainers of this project that you would suggest I am rude and unfit for the responsibilities I've been granted. I don't believe I did anything to warrant this type of treatment so I can only assume you are bitter for having your issue closed. Please understand that I did not do that to discourage you from participating in the project or attack you personally in any way so I ask that you please refrain from personal attacks like this in the future.

**Comment by julealgon** [comment]
> It may also be worth re-considering which community members are given permissions to close issues, since rude dismissals of legitimate issues are likely to deter new community members from contributing.

While I actually agreed with your concern (that it was getting more convoluted to get the project running) I think you need to reflect a bit on this particular part here.

If you go back to the closed issue, you'll notice that you called stephen out for being rude at a point up to which he had only provided you with objective content in his replies. I think he was a bit blunt and/or direct, but not rude: those are different. 

Things escalated after that.

Additionally, keep in mind that even if stephen wasn't there in the discussion, there was a strong likelyhood of the issue being closed due to the way you were exposing it. So its not a matter of stephen singling out your issue or anything like that.

Lastly, I'd say that issues being closed on the spot is a normal thing. This doesn't mean you can't counter argument and get it to be opened again. Please don't treat that as an outlandish occurrence: closing is a standard mechanism, and is sometimes used after subjectively evaluating the issue. It's totally possible that members don't see your intent the first time and assume it to be invalid, for example. All you have to do is to keep arguing, politely, for it to be reopened. Do not assume ill intent from this group: the folks here are all pretty cool, I can assure you of that ;)

Your reaction to the closure was too extreme, which led to it being locked.
